github/Repository.py

```diff
@@ -2397,6 +2397,54 @@ def protect_branch(self, branch, enabled, enforcement_level=github.GithubObject.
             headers={'Accept': 'application/vnd.github.loki-preview+json'}
         )
 
+    def remove_branch_protection(self, branch):
+        """
+        :calls: `DELETE /repos/:owner/:repo/branches/:branch/protection <https://developer.github.com/v3/repos/branches/#remove-branch-protection>`_
+        """
+        return self._requester.requestJsonAndCheck(
+            'DELETE',
+            "{base_url}/branches/{branch}/protection".format(base_url=self.url,
```

Comment:
I personally like the `format` and even f-string, but just for the sake of being consistent, would you mind changing this to the simple "+"?